### ISSUE TITLE
Synology DownloadStation: Failed Sending Torrent, Unable to bdecode torrent

### ISSUE THREAD

**Issue by Valiante** [SiCKRAGETV/sickrage-issues#1940]
**INFO**

<pre>SR Version:       BRANCH: (master) / COMMIT: (ae2c0e8006bf0c8ec3d3379148a951847b2c5f39) 
SR User:          sickrage
SR Locale:        (None, None)
SR Config file:   /usr/local/sickrage/var/config.ini
SR Database file: /usr/local/sickrage/var/sickbeard.db
SR Cache Dir:     /usr/local/sickrage/var/cache
SR Log Dir:       /usr/local/sickrage/var/Logs
SR Arguments:     ['--daemon', '--pidfile', '/usr/local/sickrage/var/sickrage.pid', '--config', '/usr/local/sickrage/var/config.ini', '--datadir', '/usr/local/sickrage/var/']
Python Version:   2.7.10 (default, Jun 3 2015, 20:43:13) [GCC 4.7.3 20130102 (prerelease)]</pre>

**ISSUE**

After installing & configuring SickRage for the first time, I'm seeing a consistent error across various search providers, so I suspect I have some kind of locale issue I don't know how to resolve, so I'm looking for some pointers if you'd be so kind.  Here's a section of log file for one such torrent;

**LOG**
<pre>2015-06-30 15:03:54 ERROR    SEARCHQUEUE-BACKLOG-264586 :: DownloadStation: Failed Sending Torrent
AA</html>
AA</body>
AA</script>
AA})();
AA       var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(sc, s);
AA       sc.src = document.location.protocol + '//s.torcache.net/sc.js';
AA      var sc = document.createElement('script'); sc.type = 'text/javascript'; sc.async = true;
AA(function() {
AA_scq.push(['load']);
AA_scq.push(['addSlot', '_47ec71d7b7be19d934c4ddbefc0aa90c']);
AA_scq.push(['setAccount', '_d0b70e33563de9bece3faa2ee3558e1e']);
AA_scq.push(['setHost', 's.torcache.net']);
AA_scq.push(['setDebug', false]);
AAvar _scq = _scq || [];
AA<script type="text/javascript">
AA<div id="_47ec71d7b7be19d934c4ddbefc0aa90c"></div>
AA	</div> 
AA		</div>
AA
AA</center>
AA	<p>You can use <a title="Magnet link" href="magnet:?xt=urn:btih:987F2719A495398C6E0E7F86C9EA0FFD737EECE6&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A80&tr=udp%3A%2F%2Ftracker.publicbt.com%3A80&tr=udp%3A%2F%2Ftracker.ccc.de%3A80" onclick="_gaq.push(['_trackEvent', 'Download', 'Magnet Link']);">magnet link</a> to download.</p>
AA<center>
AA</script>
AA    setTimeout(function () { window.location = 'http://torcache.net/torrent/987F2719A495398C6E0E7F86C9EA0FFD737EECE6.torrent'; }, 5000);
AA<script type="text/javascript" charset="utf-8">
AA<center>Your <a href="http://torcache.net/torrent/987F2719A495398C6E0E7F86C9EA0FFD737EECE6.torrent">torrent file</a> download will start in a few seconds...</center>
AA
AA</center>
AA</script>
AA  _scq.push(['addSlot', '_801a36d4e167c7184d4c696d2e103004']);
AA  var _scq = _scq || [];
AA<script type="text/javascript">
AA<div id="_801a36d4e167c7184d4c696d2e103004"></div>
AA<center>
AA<br />
AA			<h2>Torcache is a free service for caching torrent files online.</h2>
AA			<h1> Downloading... </h1>
AA		<div class="row-fluid">
AA	<div class="container-fluid">
AA	</div></div>
AA		</div>
AA			</div>
AA													</ul>
AA					<li><a href="/api"><i class="icon-wrench icon-white"></i> API</a></li>
AA									<li class="active"><a href="/"><i class="icon-home icon-white"></i> Home</a></li>
AA				<ul class="nav">
AA			<div class="nav-collapse">
AA			<a class="brand" href="/">Torcache</a>
AA		<div class="container-fluid">
AA	<div class="navbar navbar-fixed-top"><div class="navbar-inner">
AA<body>
AA	</head>
AA	<link href="http://torcache.net/css/main.css" media="all" type="text/css" rel="stylesheet">
AA<script src="http://torcache.net/bundles/bootstrapper/js/bootstrap.min.js"></script>
AA	<script src="http://torcache.net/bundles/bootstrapper/js/jquery-1.8.0.min.js"></script>
AA<link href="http://torcache.net/bundles/bootstrapper/css/bootstrap-responsive.min.css" media="all" type="text/css" rel="stylesheet">
AA<link href="http://torcache.net/bundles/bootstrapper/css/nav-fix.css" media="all" type="text/css" rel="stylesheet">
AA	<link href="http://torcache.net/bundles/bootstrapper/css/bootstrap.min.css" media="all" type="text/css" rel="stylesheet">
AA	<title>Torcache - Torrent Cache</title>
AA	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
AA<head>
AA<html lang="en">
AABTFailure: not a valid bencoded string
AA    raise BTFailure("not a valid bencoded string")
AA  File "/volume1/@appstore/sickrage/var/SickRage/lib/bencode/bencode.py", line 68, in bdecode
AA    torrent_bdecode = bdecode(result.content)
AA  File "/volume1/@appstore/sickrage/var/SickRage/sickbeard/clients/generic.py", line 157, in _get_torrent_hash
AATraceback (most recent call last):
2015-06-30 15:03:54 ERROR    SEARCHQUEUE-BACKLOG-264586 :: Unable to bdecode torrent
2015-06-30 15:03:54 INFO     SEARCHQUEUE-BACKLOG-264586 :: Downloading Orange.Is.The.New.Black.S01E01.720p.WEBRip.AAC2.0.H.264-Abjex.[e from ThePirateBay
2015-06-30 15:03:53 INFO     SEARCHQUEUE-BACKLOG-264586 :: Quality of Orange.Is.The.New.Black.S01E01.720p.WEBRip.H264-Abjex. is 720p WEB-DL
2015-06-30 15:03:53 INFO     SEARCHQUEUE-BACKLOG-264586 :: Quality of Orange.Is.The.New.Black.S01E01.720p.WEBRip.AAC2.0.H.264-Abjex.[e is 720p WEB-DL
2015-06-30 15:03:53 INFO     SEARCHQUEUE-BACKLOG-264586 :: [ThePirateBay] :: Performing episode search for Orange Is the New Black
2015-06-30 15:03:53 INFO     SEARCHQUEUE-BACKLOG-264586 :: Building internal name cache for Orange Is the New Black
2015-06-30 15:03:53 INFO     SEARCHQUEUE-BACKLOG-264586 :: Beginning backlog search for: [Orange Is the New Black]</pre>

I've tried various search providers, shows and config within SickRage, to no avail, so I suspect I'll need to dive into the shell via SSH.  I just need some advice as to what to check for / change.

**Comment by fernandog** [comment]
disable proxy (global and provider) in SR

**Comment by Valiante** [comment]
@fernandog Thanks for your reply.  I have tried with both proxy enabled and disabled, to no avail.

**Comment by Valiante** [comment]
Forgive me, but I'm not sure that other thread is the same issue, and I can't seem to see an ultimate resolution. Can you please advise what your intention is by closing this thread? Apologies if I'm missing something.

**Comment by fernandog** [comment]
@miigotu Torcache issue?

**Comment by fernandog** [comment]
@Valiante 

please do:
stop SR
edit SickRage/sickbeard/providers/generic.py
delete line 57: 'http://torcache.net/torrent/' + torrent_hash + '.torrent',
save
start SR
try again please

**Comment by Valiante** [comment]
@fernandog Thanks again for your reply.  I wasn't in a position to do this yesterday.  I went to carry out this step today and found that the one episode I'd left as "wanted" had downloaded at 5:49am.  It turns out  the OS (Synology DSM) was automatically upgraded at 5:45am.  This seems to have fixed the issue!  Sorry if I wasted any of your time.

Now I just have to figure out how to get post-processing to work.
